### ISSUE TITLE
Fashion-MNIST with tf.Keras

### ISSUE THREAD

**Issue by huhuhang** [huhuhang/ReadList#79]
**标题**：Fashion-MNIST with tf.Keras

**字数**：1006 字

**摘要**：This is a tutorial of how to classify the Fashion-MNIST dataset with tf.keras, using a Convolutional Neural Network (CNN) architecture. In just a few lines of code, you can define and train a model that is able to classify the images with over 90% accuracy, even without much optimization.

**链接**：https://medium.com/tensorflow/hello-deep-learning-fashion-mnist-with-keras-50fcff8cd74a
